python/tests/test_python_serving.py

```diff
@@ -10,35 +10,19 @@
 
 def test_server_start():
     port = random.randint(1000, 65535)
-    serving_config = ServingConfig(http_port=port,
-                                   input_data_type='NUMPY',
-                                   output_data_type='NUMPY',
-                                   log_timings=True)
+    serving_config = ServingConfig(http_port=port)
 
     python_config = PythonConfig(
         python_code='first += 2',
         python_inputs={'first': 'NDARRAY'},
         python_outputs={'first': 'NDARRAY'},
```

Comment:
@maxpumperla can we create enums here too? `VariableTypes.NDARRAY` or maybe `VariableTypes.Python.NDARRAY` and `VariableTypes.SchemaType.NDArray`